utils/generate_inference_types.py

```diff
@@ -0,0 +1,217 @@
+import argparse
```

Comment:
Missing header + maybe a mention explaining its role and the fact that it's internal only?

Reply:
similar to https://github.com/huggingface/huggingface_hub/blob/main/utils/generate_async_inference_client.py#L1-L15